### DISCUSSION TITLE
Line charts not drawing binary entities as needed

### DISCUSSION THREAD

**Discussion by damiencorpataux** [dbuezas/lovelace-plotly-graph-card#319] [Q&A] [answered]
I need to display the value of a switch in a line chart.

This works [1]:
![image](https://github.com/dbuezas/lovelace-plotly-graph-card/assets/115681/1cb50a96-9b1f-4f11-a441-f1d02579fa80)

This doesn't work:
![image](https://github.com/dbuezas/lovelace-plotly-graph-card/assets/115681/7e6a99db-a1aa-4072-b9dc-74e7bfd667a8)

So I set the `shape` of the line of my binary entity to `line`:
![image](https://github.com/dbuezas/lovelace-plotly-graph-card/assets/115681/302e30da-0bc7-4a55-8382-0eaa7680ea49)

But the line is not drawn as in [1]. Is there a specific value that I can set instead of `line` for the `shape` of the line of my binary entity ?

*Note: The context of my use case is to display a grid of graphs that are mostly `spline`, except for entities having a binary value.*

**Comment by dbuezas** [comment, marked as answer]
The right line shape in this case is hvh or hv. See all shaped available here https://plotly.com/javascript/reference/scatter/#scatter-line-shape

**Reply by dbuezas** [reply]
If you ever need to understand better the default values of things, here's where the default line shape is set in this card: https://github.com/dbuezas/lovelace-plotly-graph-card/blob/master/src/parse-config/defaults.ts#L20

**Reply by damiencorpataux** [reply]
Works like a charm !
The 2 references (plotly reference and plotly-graph defaults) are exactly what I was looking for - funny I couldn't find it.

Infinite thanks !
